attendeeguide.md

```diff
@@ -17,13 +18,14 @@ Guidelines
 
 You cannot unmute your microphone on your own. Zoom has a “raise hand” feature which you should use to alert the moderator about a question:
 
-1. Open the participants pane (we recommend doing this at the beginning of the talk). 
-2. Click the “raise your hand” button. Your name, now with a raised hand symbol, will go to the top of the participants list, which the moderator will be monitoring. 
-3. The moderator will alert the speaker, announce your name, and unmute your microphone. 
-4. Ask your question! 
-5. When the interaction ends, the moderator will mute your microphone and “lower your hand”. 
+1. Open the participants pane (we recommend doing this at the beginning of the talk).
+2. Click the “raise your hand” button. Your name, now with a raised hand symbol, will go to the top of the participants list, which the moderator will be monitoring.
+3. The moderator will alert the speaker, announce your name, and unmute your microphone.
+4. Ask your question!
+5. When the interaction ends, the moderator will mute your microphone and “lower your hand”.
 
 ### When to ask
 
 Short talks: after the talk ends.
+
```

Comment:
```suggestion
Short talks: after the talk ends.  
```

double space is a markdown newline

Reply:
Thought it looked nicer with a newline rather than a oneliner